docs/index.md

```diff
@@ -1,8 +1,8 @@
-# playing-the-internship-game
+# Playing the Internship Game
 
-Content created by [Evy Kassirer](http://www.evykassirer.com/)
+Content created by [Evy Kassirer](http://www.evykassirer.com/) and [Rudi Chen](http://digitalfreepen.com/)
 
-Also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
+You can view the source markdown files in the Docs folder. This content is also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
```

Comment:
It's like... kinda weird since I think the website homepage should be a bit different than the GitHub readme

Do you think we should consolidate or just make them more different?